core/src/Streamly/Data/MutArray.hs

```diff
@@ -69,6 +69,12 @@ module Streamly.Data.MutArray
     -- * Size
     , length
 
+    -- * Unsafe functions
+    , getIndexUnsafe
+    , putIndexUnsafe
+    , modifyIndexUnsafe
+    , writeAppendNUnsafe
```

Comment:
Is writeAppendNUnsafe being used by some code example or some user?

Reply:
No.